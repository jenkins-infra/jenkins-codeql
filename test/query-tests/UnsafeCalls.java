public class UnsafeCalls {
    public void bad() throws Exception {
        Runtime.getRuntime().exec("bad");
    }
}
