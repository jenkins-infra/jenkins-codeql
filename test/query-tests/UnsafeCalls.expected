| UnsafeCalls.java:2:17:2:19 | bad | Potentially unsafe invocation of Runtime#exec |
